<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefectWarning</class>
 <widget class="QDialog" name="DefectWarning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
   <item row="0" column="1">
    <widget class="myDefectPosLab" name="label">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>360</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>360</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Real</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>36</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>36</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mask</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>36</y>
        <width>105</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pic/image/pre_defect.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>36</y>
        <width>105</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>105</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>105</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pic/image/next_defect.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>36</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pic/image/warn.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>36</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pic/image/nowarn.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>360</width>
       <height>360</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>360</width>
       <height>360</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myDefectPosLab</class>
   <extends>QLabel</extends>
   <header location="global">mydefectposlab.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
